.gitbook/0-about-panacea/1-data-exchange-protocol/2-How-DEP-works.md

```diff
@@ -0,0 +1,69 @@
+t this time, a symmetric key based on the oracle private key is generated, so the data cannot be decrypted until the consumer also receives this symmetric key.
+After the [Incentive Distribution](../../3-protocol-devs/1-dep-specs/6-incentives.md) process is completed, the consumer can request the symmetric key from the oracle.
+The oracle checks whether the incentive distribution process is completed from the panacea blockchain, and provides the secret key encrypted based on the consumer's public key.
+As a result, it is impossible for the provider to obtain the original data before the incentive is paid, and the secret key cannot be obtained except by the consumer.
+
+## For data validation
+
+Oracle validates the data after receiving encrypted data from the provider.
+After the basic deal status, data hash validation, oracle checks whether the data satisfies the requirements specified in the deal.
+here are two types of data requirements that can be specified in the deal (json schema validation, presentation definition validation).
+
+[Json schema](https://json-schema.org) validation checks whether the data satisfies the conditions for the schema definition.
+[Presentation definition](https://identity.foundation/presentation-exchange/#presentation-definition) validation checks that the data is in the form of a did-based verifiable presentation and satisfies the presentation definition.
+This validation can additionally verify that the data was created from a verifiable credential issued by a Certificate Authority, and thus can verify that the data is authenticated by the Certificate Authority.
+
+## Atomic incentive distribution
+
+After receiving the certificate from oracle, the provider can get the incentive through [Incentive Distribution](../../3-protocol-devs/1-dep-specs/6-incentives.md) process.
+In this process, panacea blockchain checks the certificate information and immediately completes the incentive payment to the provider, and at the same time enables the consumer to obtain the secret key to decrypt the provided data.
+Thus, the provider is guaranteed to be rewarded for providing the data, and the consumer is guaranteed to access the data after completing the payment.
+
+## Decentralization & Scalability
+
+The DEP components, panacea blockchain and oracle, are open decentralized ecosystem where everyone can participate.
+Panacea blockchain is a decentralized ecosystem where anyone can participate as a full node as a public blockchain, and tx is confirmed through the consensus of validators. 
+In addition, oracle can also participate through the oracle registration process of the panacea blockchain.
+
+Panacea blockchain validators can get block generation rewards by participating in the consensus, and oracle can get commissions as incentives are paid out.
+This reward system can make a healthy decentralized ecosystem.
+
+A provider or consumer can request to any registered oracle and receive the same result.
```

Comment:
What does this mean, does it mean that any oracle is guaranteed the same response?

Reply:
I realized this was ambiguous, so I fixed it.